src/rendering/framebufferrenderer.cpp

```diff
@@ -1241,6 +1403,7 @@ const std::vector<double>& FramebufferRenderer::mSSAPattern() const {
 void FramebufferRenderer::updateRendererData() {
     ghoul::Dictionary dict;
     dict.setValue("fragmentRendererPath", std::string(RenderFragmentShaderPath));
+    dict.setValue("hdrExposure", std::to_string(_hdrExposure));
```

Comment:
I don't see how only the exposure is needed here, but not other parameters for the current TMO, such as which operator that is used, the gamma etc.